src/app/components/all-products-components/all-products-content/all-products-content.component.ts

```diff
@@ -1,16 +1,16 @@
-import {Component, Input, OnInit} from '@angular/core';
-import {ProductCard} from "../../../interfaces/product-card";
+import {Component, Input, OnInit, ViewEncapsulation} from '@angular/core';
 
 
 @Component({
   selector: 'app-all-products-content',
   templateUrl: './all-products-content.component.html',
-  styleUrls: ['./all-products-content.component.css']
+  styleUrls: ['./all-products-content.component.scss'],
+  encapsulation: ViewEncapsulation.None
 })
 export class AllProductsContentComponent implements OnInit {
 
-  @Input() products?: ProductCard[];
-
+  @Input() products?: any;
+  p: number = 3;
```

Comment:
Add access modifier for variable (public, private, protected)

Reply:
fixed